src/js/tabular_comparison/components/FilterRow.js

```diff
@@ -0,0 +1,100 @@
+import React, {useState, useCallback, useMemo} from "react";
```

Comment:
did we need to add a new FilterRow.js? can we use the one that we normally use(ui_components > filter_row.js)? also could we rename the file as `filter_row.js`?

Reply:
Done @emre2038